articles/multifactor-authentication/api/manage.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Manage the Authenticators for Multifactor Authentication
+description: How to manage your MFA authenticators
+beta: true
+---
+
+::: warning
+This article includes documentation on features that are still under development. These features are available to customers with early access.
+:::
+
+# Manage the Authenticators
+
+Auth0 provides several API endpoints to help you manage the authenticators you're using with a particular tenant for multifactor authentication (MFA).
+
+## List Authenticators
+
+To get a list of the authenticators you've associated and can be used with your tenant, you can make the appropriate call to the `/mfa/authenticators` endpoint:
+
+```har
+{
+	"method": "GET",
+	"url": "https://${account.namespace}/mfa/authenticators",
+	"headers": [{
+		"name": "Authorization",
+		"value": "Bearer YOUR_API_ACCESS_TOKEN"
+	}]
+}
+```
+
+You should receive information about the authenticator type(s) in the response:
+
+```json
+[
+  {
+    "authenticator_type": "recovery-code",
+    "id": "recovery-code|dev_IsBj5j3H12VAdOIj"
```

Comment:
We also return an `"active": true` or `"active": false` field here to indicate if the authenticator has been confirmed.